packages/patternfly-4/react-core/src/components/Pagination/Navigation.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly/components/Pagination/pagination.css';
+import { css } from '@patternfly/react-styles';
+import { AngleLeftIcon, AngleDoubleLeftIcon, AngleRightIcon, AngleDoubleRightIcon } from '@patternfly/react-icons';
+import { Button, ButtonVariant } from '../Button';
+
+const propTypes = {
+  lastPage: PropTypes.number,
+  titles: PropTypes.shape({
+    pages: PropTypes.string
+  }),
+  page: PropTypes.number.isRequired,
+  onSetPage: PropTypes.func.isRequired,
+  onNextclick: PropTypes.func,
+  onPreviousClick: PropTypes.func,
+  onFirstClick: PropTypes.func,
+  onLastClick: PropTypes.func
+};
+const defaultProps = {};
+
+const Navigation = ({
+  page,
+  lastPage,
+  titles,
+  onSetPage,
+  onNextClick,
+  onPreviousClick,
+  onFirstClick,
+  onLastClick,
+  className,
+  ...props
+}) => {
+  return (
+    <nav className={css(styles.paginationNav, className)} aria-label="Pagination" {...props}>
```

Comment:
I don't think that it's a good idea to add specific aria-label prop, cause it would increase the number of props in prop table. If consumers want to add aria-label they probably know why they want it to be thee.

Reply:
I'm not sure how this should be implemented, but it would be great if we could support the following use cases:

- By default, `aria-label` is set to "Pagination". This is a standard string we're recommending for this component. We also don't expect consumers to realize that this attribute is needed here, which is why we'd like to include something by default.
- If consumers want to provide translated strings for the product, they should be able to override the default value with their translated string. 

In other words, we'd like our components to be accessible by default without consumers having to do much, but we also want the flexibility to allow consumers to replace default text strings.

Reply:
@jgiardino that's exacly what is implemente here. If consumer didn't pass any `aria-label` to this component it will default to `Pagination` however if they are using some tranaslation strings or want to pass something specific they can choose to add their own attribute and it will be overwritten by their string.